package.json

```diff
@@ -1,29 +1,29 @@
 {
   "name": "datahub-link-generator",
-  "version": "3.0.0",
-  "description": "",
+  "version": "1.0.0",
```

Comment:
This downgrades the version from 3.0.0 to 1.0.0. Is that what you intended?

Reply:
No, I just upgraded the version to 4.0.0. Thanks for the catch.